source/application.txt

```diff
@@ -49,7 +49,7 @@ When creating your application it takes a flag property of :class:`Gio.Applicati
 Using this you can let it handle everything itself or have more custom behavior.
 
 You can use `HANDLES_COMMAND_LINE` to allow custom behavior in :meth:`Gio.Application.do_command_line()`.
-In combination with :meth:`Gio.Application.add_main_option()` to add custom options.
+In combination with :meth:`Gio.Application.add_main_option_entries( <array of GLib.OptionEntry() objects> )` to add custom options.
```

Comment:
https://valadoc.org/gio-2.0/GLib.Application.add_main_option.html is "since 2.42" and 
https://valadoc.org/gio-2.0/GLib.Application.add_main_option_entries.html is "since 2.40" and will make a whole lot of people less frustrated that their install does not run the example code.

Reply:
I fail to see why the example should be made worse to care about a version nobody uses.

The only relevant distro from that period is Ubuntu 14.04 which is too old for 2.40 anyway.

Reply:
What do each of those arguments mean? You can see what is going to the function and each OptionEntry attribute is clearly marked without digging for docs. Plus it works and does not crash if you have earlier libraries.

Reply:
The only advantage I can see is that `add_main_option_entries` allows adding multiple entries at the same time. Since both are relevant, you should mention both of same.